deploy/likedao/templates/bdjuno.config.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bdjuno-config-{{ .Values.deploymentTag }}
+data:
+  config.yaml: |
+    chain:
+        bech32_prefix:
+            {{- range .Values.bdjuno.config.chain.bech32Prefix }}
+              - {{ . }}
+            {{- end}}
+        modules:
+            {{- range .Values.bdjuno.config.chain.modules }}
+              - {{ . }}
+            {{- end}}
```

Comment:
I don't think we will config modules?

Reply:
it's possible for other chains to add in their own modules, we will need to add `iscn` to this field when it's ready as well

Reply:
I think we require some moudles in order for dao to work?

Let's make it only for addiational modules, not listing all.